### DISCUSSION TITLE
speed.cloudflare.com 的流量限制是全局的还是分IP的？

### DISCUSSION THREAD

**Discussion by CrazyBoyFeng** [XIU2/CloudflareSpeedTest#175] [提问]
似乎 #168 中包括不少人认为是全局的？

但是根据我的经验，当API出现返回503后，我重新拨号换个IP就可以继续测速了。
如果是分IP限制，那么这个测速API大多数情况下都能满足需求，可以替代自建API。

**Comment by XIU2** [comment]
不知道呀，我印象中，大概一年多以前，我这个软件当时的默认下载测速地址就是 `https://speed.cloudflare.com/__down?bytes=300000000` 只是后来一些人反馈一到晚上就无法测速了，我试了下确实是这样，浏览器访问这个 URL 会提示流量超限之类的话，但平时我自己是很少下载测速的，应该是全局的。

**Reply by CrazyBoyFeng** [reply]
那我早上7点多试试。  
cf的重置时间是utc的0点，也就是北京时间8点。

**Reply by XIU2** [reply]
@CrazyBoyFeng 现在用的人不多，应该还没有超出总流量限制，我自己写的有脚本，用来每分钟更换重定向目标，并且还会在每天 23 点、7 点的时候检查一遍所有可用地址状态，目前这几天都没有发出失效警报。
